Source/WebCore/platform/mock/mediasource/MockSourceBufferPrivate.cpp

```diff
@@ -107,7 +107,7 @@ class MockMediaDescription final : public MediaDescription {
     static Ref<MockMediaDescription> create(const MockTrackBox& box) { return adoptRef(*new MockMediaDescription(box)); }
     virtual ~MockMediaDescription() = default;
 
-    AtomString codec() const override { return m_box.codec(); }
+    AtomString codec() const override { return AtomString { m_box.codec() }; }
```

Comment:
I think we should change `MockTrackBox` instead, but not critical since this is only for testing.

Reply:
Yes, fixing.